optimum/habana/transformers/models/mpt/modeling_mpt.py

```diff
@@ -25,10 +25,40 @@
 from transformers.utils import logging
 
 from ...modeling_attn_mask_utils import _gaudi_prepare_4d_causal_attention_mask
+try:
+    from habana_frameworks.torch.hpex.kernels import FusedSDPA
+except ImportError:
+    print("Not using HPU fused scaled dot-product attention kernel.")
+    FusedSDPA = None
 
+import habana_frameworks.torch.core as htcore
 
 logger = logging.get_logger(__name__)
 
+use_custom_sdpa=False
+# Efficient implementation equivalent to the following:
```

Comment:
Thank you for your suggestion:). The alibi bias used by mpt model is header-dependent, but 'habana_frameworks.torch.hpex.kernels.FusedSDPA' is very likely to only support header-independent bias. We are asking the habana team for help. This PR can be put on hold for now. If there is any progress, I will update the code. Otherwise, the MPT model does not support FusedSDPA.

Reply:
I see, thanks for explaining. If you'd like to pause review on the PR, please put it in "Draft" state. When you resume working on it, you can put it back to "Ready for Review". Thanks!